backend/src/middleware/notifications/notificationsMiddleware.js

```diff
@@ -1,6 +1,8 @@
 import extractMentionedUsers from './mentions/extractMentionedUsers'
 import { validateNotifyUsers } from '../validation/validationMiddleware'
 
+const debug = require('debug')('human-connection-backend:notificationsMiddleware')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-01T14:17:59Z" title="Saturday, February 1st 2020, 3:17:59 pm +01:00">Feb 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,8 @@
 import extractMentionedUsers from './mentions/extractMentionedUsers'
 import { validateNotifyUsers } from '../validation/validationMiddleware'
 
+const debug = require('debug')('backend:notificationsMiddleware')
```


```suggestion
const debug = require('debug')('human-connection-backend:notificationsMiddleware')
```
This should be the `name` in our `package.json` as per [convention](https://www.npmjs.com/package/debug#conventions). It's not like this everywhere in the code base because we recently changed the `name` (could be refactored, too).

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T13:17:42Z" title="Wednesday, February 5th 2020, 2:17:42 pm +01:00">Feb 5, 2020</time>_

---


Ah okay. @roschaefer 
Have saved that as a reminder. May I will create an issue …

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T13:17:56Z" title="Wednesday, February 5th 2020, 2:17:56 pm +01:00">Feb 5, 2020</time>_

---


Done by the way 👍🏼